### ISSUE TITLE
Compiling with MSVC 2014 causes error C2280: 'synergy::KeyMap::Keystroke::Data::Data(void)': attempting to reference a deleted function

### ISSUE THREAD

**Issue by AnrDaemon** [symless/synergy-core#6956]
### Operating Systems ###

Windows 7 (but irrelevant, see below)

### Synergy Version ###

1.13.1.41-stable

### Steps to Reproduce ###

Compiling the tag with

```cmd
@ECHO OFF

SET CMAKE_PREFIX_PATH=C:\Qt\5.12.5\msvc2015_64
SET BUILD_CONFIG=MinSizeRel

MKDIR "%~dp0build"

CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat"

(
CD /D "%~dp0build"

cmake -G "Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=%BUILD_CONFIG% ..

msbuild synergy-core.sln /p:Platform="x64" /p:Configuration=%BUILD_CONFIG% /m

COPY "%~dp0ext\openssl\windows\x64\bin"\* bin\%BUILD_CONFIG%\
) > cmake.log 2> cmake.err
```

causes error

```
"…\synergy-core\build\synergy-core.sln" (целевой объект по умолчанию) (1) ->
"…\synergy-core\build\src\test\integtests\integtests.vcxproj.metaproj" (целевой объект по умолчанию) (15) ->
"…\synergy-core\build\src\lib\synergy\synlib.vcxproj.metaproj" (целевой объект по умолчанию) (30) ->
"…\synergy-core\build\src\lib\synergy\synlib.vcxproj" (целевой объект по умолчанию) (31) ->
(Целевой объект ClCompile) -> 
  …\synergy-core\src\lib\synergy\KeyMap.cpp(1330): error C2280: 'synergy::KeyMap::Keystroke::Data::Data(void)': attempting to reference a deleted function […\synergy-core\build\src\lib\synergy\synlib.vcxproj]
```

### Log URLs ###

[cmake.log](https://github.com/symless/synergy-core/files/6078464/cmake.log)

**Comment by andreacndn** [comment]
SYNERGY-854

**Comment by AnrDaemon** [comment]
Still an issue with 1.13.2.20-snapshot

**Comment by vitalPixel** [comment]
This should now be fixed with the release of Synergy 1.14.2.

If you're experiencing this problem please make sure you are running Synergy latest version (you can download it by logging in to your account here: https://symless.com/synergy/download).

Please let us know if you run into any issues.

**Comment by vitalPixel** [comment]
This should now be fixed with the release of Synergy 1.14.2.

If you're experiencing this problem please make sure you are running Synergy latest version (you can download it by logging in to your account here: https://symless.com/synergy/download).

Please let us know if you run into any issues.

**Comment by AnrDaemon** [comment]
Seems all right with 1.14.2.36

Logs are clear and binaries correctly produced in the destination directory.
[cmake.log](https://github.com/symless/synergy-core/files/7649983/cmake.log)
